docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
```

Comment:
Ok so let me finish other parts like docs first and I will squeeze lets encrypt in. 
Yes in any case you have to modify configs and paths. But taking out lets encrypt cert generation etc is much more to do for those who have own certs. But I agree that if the private user installs this we should give him highest security. We e.g. can have a secure setup where the user has to care for his cert, but all is set to SSL and https. I think its more about comfort and automation ?